app/models/user.rb

```diff
@@ -14,4 +14,36 @@ class User < ApplicationRecord
   def email_downcase
     self.email = email.downcase
   end
+
+  attr_accessor :remember_token
+
+  class << self
+    def User.digest string
+      cost = if ActiveModel::SecurePassword.min_cost
+               BCrypt::Engine::MIN_COST
+             else
+               BCrypt::Engine.cost
+             end
+      BCrypt::Password.create string, cost: cost
+    end
+
+    def new_token
+      SecureRandom.urlsafe_base64
+    end
+  end
+
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
```

Comment:
```suggestion
    update_column :remember_digest, User.digest(remember_token)
```
update_column hợp lý hơn về hiệu suất nhé